test/assets/unrunnableTestCases.csv

```diff
@@ -4197,5 +4197,4 @@ fo-test-array-subarray-002,Error: FOAY0001: subarray start out of bounds.
 fo-test-array-subarray-006,Error: FOAY0001: subarray start out of bounds.
 fo-test-array-subarray-007,Error: FOAY0001: subarray start out of bounds.
 fo-test-array-fold-right-003,AssertionError: Expected XPath          array:fold-right([1,2,3], [], function($x, $y){[$x, $y]})        to (deep equally) resolve to [1, [2, [3, []]]]: expected false to be true
-fo-test-array-sort-002,Error: XPST0017: Function array:sort with arity of 3 not registered. Did you mean "Q{http://www.w3.org/2005/xpath-functions/array}sort (array(*))", "Q{http://www.w3.org/2001/XMLSchema}short (xs:anyAtomicType?)" or "Q{http://www.w3.org/2005/xpath-functions/math}sqrt (xs:double?)"?
-value-comparison-12,AssertionError: Expected XPath //test:comparable[1]/test:int eq //test:comparable[2]/test:int to resolve to true: expected false to be true
\ No newline at end of file
+fo-test-array-sort-002,Error: XPST0017: Function array:sort with arity of 3 not registered. Did you mean "Q{http://www.w3.org/2005/xpath-functions/array}sort (array(*))", "Q{http://www.w3.org/2001/XMLSchema}short (xs:anyAtomicType?)" or "Q{http://www.w3.org/2005/xpath-functions/math}sqrt (xs:double?)"?
```

Comment:
?

Reply:
I guess that's fonto's unRunnableTest

Reply:
One test from unrunnableTestCases got removed nothing was added to this file.